html/_sources/index.txt

```diff
@@ -1,13 +1,13 @@
-SymPy Tutorial for PyCon 2015
+SymPy Tutorial for PyDelhiConf 2016
 =============================
 
-This is the tutorial will be presented at PyCon India 2015 by SymPy India
+This is the tutorial will be presented at PyDelhiConf India 2016 by SymPy India
```

Comment:
This can be improved: `This is the tutorial will be presented a` -> `This tutorial will be presented at`